tests/post/features/pods_alive.feature

```diff
@@ -0,0 +1,6 @@
+@liveness @post @ci @local
+Feature: Pods should be alive
+    Scenario: get pods
+        Given The bootstrap phase is executed
```

Comment:
This is implemented as a no-op. Will that remain the case? Shouldn't it somehow be an assertion at least?